Doc/c-api/stable.rst

```diff
@@ -66,7 +66,7 @@ Limited C API
 
 Python 3.2 introduced the *Limited API*, a subset of Python's C API.
 Extensions that only use the Limited API can be
-compiled once and work with multiple versions of Python.
+compiled once and be loaded on multiple versions of Python.
```

Comment:
It would be better to use `ABI Compatible` here instead of `loaded on` as the latter means that extension module can be loaded which may not be the case if the extension uses any function which now raises error while importing.

Reply:
Thanks for the suggestion.
This intro is meant to communicate the intent of the limited API, rather than give the precise description. I'd rather leave the term “ABI Compatible” to the sections below, which should also clear up *which* “multiple versions” the extension will work with.